templates/js/models/sequelize/config/config.js

```diff
@@ -1,15 +1,10 @@
-import dotenv from 'dotenv';
-dotenv.config();
-
-const envVars = process.env;
-
 const config = {
-  username: envVars.USERNAME,
-  password: envVars.PASSWORD,
-  database: envVars.DATABASE,
-  host: envVars.HOST,
-  port: envVars.DB_PORT,
-  dialect: envVars.DIALECT,
+  username: process.env.USERNAME,
```

Comment:
for whatever reason trying to access this either destructured or by assigning `envVars` to `process.env` broke and all these were undefined.

Reply:
weird. but its fine.

Reply:
yeah i thought the plugin was broken, tried 2 other babel plugins. When the 3rd plugin failed i realized there must be an error in our code not all 3 npm packages :trollface: